### PR TITLE
Fix metrics cryptocurrency to u64

### PR DESCRIPTION
**Pull request by tworec**
fixes: #613

### CODE REVIEW

**Review by jiivan — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -137,6 +137,9 @@ members = [
 ]
 
 [patch.crates-io]
+## EXTERNAL
+actix-web = { git = "https://github.com/tworec/actix-web.git", branch = "middleware-logger-debug-level"  }
+
```
Comment (jiivan) [inline Cargo.toml:142]: Looks unrelated

File: core/market/decentralized/src/negotiation/requestor.rs [hunk 2]
```diff
@@ -473,7 +473,7 @@ pub async fn proposal_receiver_thread(
         let db = db.clone();
         let notifier = notifier.clone();
         match async move {
-            log::info!("Received proposal from matcher. Adding to events queue.");
+            log::info!("Got matching Offer. Emitting new Proposal to Requestor.");
```
Comment (jiivan) [inline core/market/decentralized/src/negotiation/requestor.rs:476]: Unrelated

**Review by jiivan — CHANGES_REQUESTED** [review]

**Review by nieznanysprawiciel — APPROVED** [review]

**Review by jiivan — APPROVED** [review]

**Review by maaktweluit — APPROVED** [review]
